nba/data_load.py

```diff
@@ -1,13 +1,14 @@
 import pandas as pd
 from pathlib import Path
 
-
 def load_csv(data_path: Path) -> pd.DataFrame:
     return pd.read_csv(data_path)
 
-
 if __name__ == "__main__":
-    # TODO: fix this data path using the pathlib library
-    data_path = "/Users/adrianochs/all_my_great_data/game.csv"
+    # Define the base directory dynamically (assumes the CSV is within the current script's directory)
+    base_dir = Path(__file__).parent
```

Comment:
Hello, 
Path(__file__).parent is pointing at the nba folder, I think we are supposed to go one more level up to the PS2_EX4_CONFIG folder -> 
 base_dir = Path(__file__).parent.parent

Reply:
fixed! thanks for the flag